src/components/callout/callout.helpers.ts

```diff
@@ -0,0 +1,40 @@
+import translate from '../../global/translations';
+
+const calloutIcons: Record<string, string> = {
+    danger: 'poison',
+    info: 'info',
+    note: 'appointment_reminders',
+    tip: 'idea',
+    warning: 'error',
+    success: 'ok',
+    failure: 'high_priority',
+    example: 'test_tube',
+    quote: 'quote_left',
+    question: 'help',
+};
+
+export function getIcon(icon: string, type: string = 'info'): string {
+    if (icon) {
+        return icon;
+    }
+
+    return calloutIcons[type] ?? calloutIcons.info;
+}
+
+export function getHeading(
+    heading: string,
+    type: string = 'info',
+    language: string = 'en'
+): string {
+    if (heading) {
+        return heading;
+    }
+
+    const key = `callout.${type}`;
+
+    try {
+        return translate.get(key, language);
+    } catch (error) {
+        return translate.get(key, 'en');
+    }
```

Comment:
I did this because the component breaks if a language is given that doesn't have a translation.
So I will catch that error and default it to english.

I'm not sure that this component should handle this, maybe it would be better to have it in the translation function.
But this works and won't affect any other components, so that's why I did it